docs/pages/reference/config.mdx

```diff
@@ -484,6 +484,24 @@ auth_service:
     # See "TLS Routing" in Architecture section for additional information.
     proxy_listener_mode: multiplex
 
+    # Determines the strategy that the cluster uses for connecting clients to
+    # agents in IoT mode.
```

Comment:
I think we should say "through the Proxy Service" instead of "in IoT mode". `iot mode` is hardly mentioned at all in the docs, and with Teleport Cloud, resource services/agents/etc. in non-IoT use cases still connect to the Auth Service via the Proxy Service.